pages/in/widget/iscn-ar/index.vue

```diff
@@ -265,6 +265,8 @@ export default {
       fingerprints: [],
       title: '',
       type: 'article',
+      author: '',
+      description: '',
```

Comment:
Is this description `description` as in iscn contentMeta description, or author's description?
If you name it as `description` I would think its iscn's descrption
same for all code below

Reply:
I think it was originally the description for ISCN. But, in WordPress context, it is description for author.
Hence, need to discuss with product owner, what / how to show.

Reply:
will separate ISCN description & author description